src/pages/PeopleTabPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const PeopleTabPage: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isLoadin, setIsLoading] = useState(false);
```

Comment:
Be careful when naming, there is a typo in the variable name `isLoadin`. It should be `isLoading`.